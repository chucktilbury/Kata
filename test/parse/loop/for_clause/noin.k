;-------------------------------------------------------------------------------
; Demonstrate no 'in' keyword
;-------------------------------------------------------------------------------

func name ()() {

    for(a_name comp.name + 1) {}
}
