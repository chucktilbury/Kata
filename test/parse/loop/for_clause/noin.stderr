Syntax: /home/chuck/Src/Kata/test/parse/loop/for_clause/noin.k: 7: 20: expected the 'in' keyword but got symbol
Syntax: /home/chuck/Src/Kata/test/parse/loop/for_clause/noin.k: 7: 20: expected function body element but got for
Syntax: /home/chuck/Src/Kata/test/parse/loop/for_clause/noin.k: 7: 20: expected function body but got for
Syntax: /home/chuck/Src/Kata/test/parse/loop/for_clause/noin.k: 7: 20: expected module item or end of file but got func
Syntax: /home/chuck/Src/Kata/test/parse/loop/for_clause/noin.k: 7: 20: module cannot be empty
